js_modules/dagster-ui/packages/ui-core/src/assets/auto-materialization/AutomaterializationEvaluationHistoryTable.tsx

```diff
@@ -86,7 +86,11 @@ export const AutomaterializationEvaluationHistoryTable = ({
                   <Timestamp timestamp={{unix: tick.timestamp}} timeFormat={{showTimezone: true}} />
                 </td>
                 <td>
-                  <TickStatusTag tick={tick} isStuckStarted={isTickStuckInStartedState} />
+                  <TickStatusTag
+                    tick={tick}
+                    tickResultType="materializations"
```

Comment:
Previously, we were detecting whether to show "materializations requested" or "runs requested" based on the fields loaded in `tick`. This doesn't work anymore because we're showing both styles of sensor in the same UI.

I talked with Owen and there isn't an attribute of the tick we can use to determine what the appropriate style is -- it's based on whether the sensor emitting the tick is an automation sensor.  I think that explicitly passing the rendering desired is best but it did make this PR a bit beefier...